chef_master/source/supermarket_backup_restore.rst

```diff
@@ -3,21 +3,86 @@ Supermarket Backup and Restore
 =====================================================
 `[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/supermarket_backup_restore.rst>`__
 
-Periodic backups of Supermarket data are an essential part of managing and maintaining a healthy configuration and to help ensure that important data can be restored, if required.
+Periodic backups of Supermarket data are an essential part of managing and maintaining a healthy configuration, and to help ensure that important data can be restored if required. In a typical installation of Supermarket, both the cookbook store and the database need to be backed up on a regular basis.
 
-**Cookbooks**
 
-The default location of cookbooks is here: `/var/opt/supermarket/data`
+Backup
+=====================================================
+
+Cookbook Backup
+-----------------------------------------------------
+
+If you are not using AWS S3 storage for cookbooks, you will also need to make a backup of the local cookbook storage location. 
+
+The default location is: ``/var/opt/supermarket/data/cookbook_versions``.
+
+For example, a cookbook backup command:
+
+.. code-block:: bash
+
+     cd /var/opt/supermarket/data/ 
+     tar cvzf supermarket_cookbook_versions.tar.gz cookbook_versions
```

Comment:
I recommend not putting the resulting backup tar.gz file in the `/var/opt/supermarket/data/ ` directory. Maybe this way, which will place it in the user's home directory:

```suggestion
     tar cvzf ~/supermarket_cookbook_versions.tar.gz cookbook_versions
```